modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -49,3 +49,31 @@ Specify `false` if you want explicitly request an IP address from this pool with
 The default value is `true`.
 
 |===
+
+You can assign IP addresses from an `IPAddressPool` to services and namespaces by configuring the `spec.serviceAllocation` specification.
+
+.MetalLB IPAddressPool custom resource spec.serviceAllocation subfields
+[cols="1,1,3a", options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`priority`
+|`int`
+|Optional: Defines the priority between address pools when more than one address pool matches a service or namespace. A lower number equals a higher priority.
```

Comment:
Would you want call address pools as ''ip address pools' instead to reinforce it is 'IPAddressPool' CR versus 'Addresspool' CR.

Reply:
Good catch, thanks.